### PR TITLE
Rewrite CoC

### PR DESCRIPTION
**Pull request by shymega**
This PR (28 commits, was done over long period of time), rewrites the Code of Conduct (ignore coc.md, testing file), to be less authoritative, and to address several issues that have arisen since the last revision of the main CoC.

### CODE REVIEW

**Comment by vktec** [comment]
Looks good, I'd be happy for this to be merged. @zleap any thoughts?

File: CONDUCT.md [hunk 1]
```diff
@@ -1,84 +1,100 @@
...
-    answer your question or guide you to an answer, to the best of our ability!
-  - Make friends! vKTec IRC is a place to meet and talk with like-minded people
-    so find some channels and chat to people!
-
-### Please do not
-
-At vKTec IRC we like to have a laid back atmosphere but in order to do so, we
-need to set a few rules :)
-
-  - Don't say or do anything you wouldn't mind everyone on the internet being
```
Comment (zleap) [inline CONDUCT.md]: you have Please do not at the top of this section,  and are then putting Don't in the list,  isn't this a double negative.  line 28,  and others

File: CONDUCT.md [hunk 2]
```diff
@@ -1,84 +1,100 @@
...
+Welcome to the vKTec IRC network!
+
+By connecting to this network, we expect you to read below for what we
+expect of our users:
+
+> IRC is a privilege, not a right.
+
+# Etiquette
+
+- Don't speak in capitals. This is seen as rude, and unnecessary.
```
Comment (zleap) [inline CONDUCT.md]: I still think putting don't at the start is too negative,  have something like
Speaking in capitals is seen as shouting and therefore rude, please use lower case letters and perhaps use _to_ or *to*  emphasize a point.   Capitals are depreciated on IRC generally
 (clearly the exception to this is using acronyms or abbreviations. )

File: CONDUCT.md (hunk 1 above)
Comment (vktec) [inline CONDUCT.md]: @zleap exactly. This is the bit that's being removed. It's a diff, not the whole file.

**Comment by vktec** [comment]
I think this is ready to merge, so I'll go ahead and do that now. *braces for merge conflicts*

**Comment by shymega** [comment]
Alright then

**Pull request merged**